view_output.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
-filename="$(find . -name '*.pdf')"
+filename="$1"
 
 if type xdg-open > /dev/null 2>&1; then
     # X11-based systems (Linux, BSD).
     
-    if [ $1 = "None" ] ; then   
+    if [ $2 = "None" ] ; then
```

Comment:
Here might be a good idea to name the variable, and also quote it

Reply:
Done.